### DISCUSSION TITLE
No code completion proposals

### DISCUSSION THREAD

**Discussion by freggern** [alagrede/znote-app#71] [General]
When I work inside the JS snipped I don't get any code completion proposals. Maybe a windows build problem. When I type something there is a delay.

**Comment by alagrede** [comment]
Hi @freggern,
Thank you for your reporting! Seems there is a performance issue with the Windows build. Working on it!
Thanks

**Reply by alagrede** [reply]
Hi @freggern,
After investigating my Windows X64 machine this morning, it seems that the typing time is so long...
As a reminder: ARM Windows is not currently well supported.

I just add in **V1.7.10** new settings for you to set the refresh time and set a default preview refresh time less aggressive for Windows.
If you are still having performance issues, try closing the preview.

For the completion, I don't know why you haven't suggestions with `CMD+Space`. Do you have an **arm** or **x64** Windows?
Might be interested to know.

<img width="621" alt="settings-refresh" src="https://user-images.githubusercontent.com/5312754/200564222-1c86033b-aa5b-4e97-90d4-7a78d1cece3a.png">

Hope this will help you.

Anthony

**Reply by freggern** [reply]
Hello @alagrede,

Thanks for the quick performance fix. The typing delay looks good. Also, suggestions work. I assumed that suggestions are always displayed (video on the main page misled me) and I didn't need to click ctrl+space. Is there a way or is it planned in the future to always display suggestions when typing?
